<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>143</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>300</number>
      </attribute>
      <row>
       <property name="text">
        <string> 1 </string>
       </property>
      </row>
      <row>
       <property name="text">
        <string> 2 </string>
       </property>
      </row>
      <row>
       <property name="text">
        <string> 3 </string>
       </property>
      </row>
      <row>
       <property name="text">
        <string> 4 </string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Search</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Replace</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string>(w:right=&quot;)\d+</string>
       </property>
      </item>
      <item row="0" column="1">
       <property name="text">
        <string>$10</string>
       </property>
      </item>
      <item row="1" column="0">
       <property name="text">
        <string>(pgSz w:w=&quot;)\d+(&quot; w:h=&quot;)\d+</string>
       </property>
      </item>
      <item row="1" column="1">
       <property name="text">
        <string>$111907$216840</string>
       </property>
      </item>
      <item row="2" column="0">
       <property name="text">
        <string>(pgMar w:top=&quot;\d+&quot; w:right=&quot;)\d+(&quot; w:bottom=&quot;\d+&quot; w:left=&quot;)\d+</string>
       </property>
      </item>
      <item row="2" column="1">
       <property name="text">
        <string>$1851$21134</string>
       </property>
      </item>
      <item row="3" column="0">
       <property name="text">
        <string>(&lt;o:Revision&gt;1&lt;/o:Revision&gt;)</string>
       </property>
      </item>
      <item row="3" column="1">
       <property name="text">
        <string>&lt;o:Keywords&gt;A4&lt;/o:Keywords&gt;$1</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_BrowseXML"/>
   <addaction name="action_About"/>
   <addaction name="action_Exit"/>
  </widget>
  <action name="action_Exit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="statusTip">
    <string>Exit App</string>
   </property>
   <property name="shortcut">
    <string>F10</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
   <property name="priority">
    <enum>QAction::NormalPriority</enum>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="statusTip">
    <string>About App</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="action_BrowseXML">
   <property name="text">
    <string>S&amp;elect XML</string>
   </property>
   <property name="statusTip">
    <string>Open and Save XML</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
